content/documentation.rst

```diff
@@ -256,6 +262,19 @@ Further details about generating and building documentation is found in the
 chevah server repository.
 
 
+Communicating command-line syntax
+----------------------------------
+
+Use the following convention:
+
+.. sourcecode:: bash
+
+    $ client-shell webdavs://user@acme.onmicrosoft.com@acme.sharepoint.com -p 'password'
+    > connect
+
+$ means a non-root user, # is a root user and > means a client-shell command.
```

Comment:
Maybe we shoud use ` ``$`` means something`

also it would help to also have an example for root commad like

```
# useradd sftpplus
```